pkg/action/build-uki.go

```diff
@@ -212,6 +219,14 @@ func (b *BuildUKIAction) checkDeps() error {
 		}
 	}
 
+	if b.arch == "x86_64" || b.arch == "amd64" {
```

Comment:
imo this should be a function from the `buildUKIAction`

Reply:
good point!